opentreemap/treemap/instance.py

```diff
@@ -573,30 +572,16 @@ def update_revs(self, *attrs):
         for attr in attrs:
             setattr(self, attr, getattr(qs[0], attr))
 
-    def itree_region_codes(self):
-        from treemap.models import ITreeRegion
+    def itree_regions(self, **extra_query):
```

Comment:
(summary of our discussion)
The list vs queryset ducktyping is idiomatic python. While there are not other places in the app we use ducktyping in this fashion, it is still a useful solution for encapsulating the data model of itree regions. There is also not a big performance worry, because ITreeRegion data is small, and these methods don't deal with the geom field.

I have expanded the commit message to explain why I think this is better.